packages/flutter_tools/lib/src/dart/analysis.dart

```diff
@@ -46,6 +46,7 @@ class AnalysisDriver {
   DriverOptions options;
 
   String get sdkDir {
+    // ignore: deprecated_member_use
```

Comment:
This works, but unless we're relying on being able to override the sdk location via an env variable or other features, we could just change sdkDir to:

```
String get sdkDir => clu_util.getSdkPath();
```